### ISSUE TITLE
Missing DLL crashes 10x

### ISSUE THREAD

**Issue by kayakakan** [slynch8/10x#1072]
Windows 7, 10x can't launch because api-ms-win-core-path-l1-1-0.dll is missing. I happen to have that dll. I got it from [https://github.com/nalexandru/api-ms-win-core-path-HACK](url). Adding it to the root installation of 10x resolves the issue. I believe it's a requirement from python, and that the dll was introduced in Windows 8. I'll let you decide on the course of actions from these informations, and hope it doesn't lead to a drop of the support for Windows 7. Thank you for your work regardless of your choice.

**Comment by slynch8** [comment]
It's difficult to know what to do about this. If I include that dll then it may break for non-Win7 users. Glad you got it working though. I'll try and keep Win7 working as much as possible.

**Comment by kayakakan** [comment]
Yeah, having multiple versions of a dll in a computer doesn't seem like a good idea. Is there a way to let the knowledge on how to keep a Windows 7 version working, clearly visible for future users, like in an FAQ? I would be okay to have the issue closed for now if that were the case.

**Comment by slynch8** [comment]
Yes, I'll keep this issue open to remind me to add documentation.

**Comment by kayakakan** [comment]
Thank you, that's enough for me. That'll teach me to use a _deprecated_ OS.
